code/modules/paperwork/desk_bell.dm

```diff
@@ -0,0 +1,83 @@
+// A receptionist's bell
+
+/obj/structure/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	layer = OBJ_LAYER
+	anchored = FALSE
+	pass_flags = PASSTABLE // Able to place on tables
+	max_integrity = 5000 // To make attacking it not instantly break it
+	/// The amount of times this bell has been rang, used to check the chance it breaks
+	var/times_rang = 0
+	/// Is this bell broken?
+	var/broken_ringer = FALSE
+	/// Last time the bell was rung, needed for paradise port.
+	var/last_ring = 0
+	/// The length of the cooldown. Setting it to 0 will skip all cooldowns alltogether.
+	var/ring_cooldown_length = 0.3 SECONDS // This is here to protect against tinnitus.
```

Comment:
Ring sound is var-editable, so it'd be good that admins can change it (specifically to increase it so a longer sound doesn't stack on itself and destroy people's ears.)

 There was also a speeddemon variant that this was written for with no limit, but I removed that because I didn't see a point in something only existing to let you earrape someone with a mouse macro.